src/main/java/com/exasol/bucketfs/BucketAccessException.java

```diff
@@ -7,7 +7,9 @@
  */
 public class BucketAccessException extends Exception {
     private static final long serialVersionUID = -1002852289020779835L;
```

Comment:
Do you still need the serialVersionID?

Reply:
I think you should have it fo all serializable classes. So probably yes. I only added the javadoc tags, since they are required by the new template of project-keeper.